packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -68,6 +80,11 @@ export const connectEpic: Epic<AnyAction, AnyAction, { ws: RootState }, any> = (
       const isConnecting = state.ws.connections.all[connectionKey]?.connecting > 1
       return !isActiveConnection && !isConnecting
     }),
+    concatMap(async (data) => {
+      const url = data[0].payload.wsHandler.connection.url
+      if (typeof url !== 'string') data[0].payload.wsHandler.connection.url = await url()
```

Comment:
Mind adding a comment here for when url can be a function? At a glance it reads as-if url should always be a string property, otherwise it would be named `getUrl()`. How do you feel about `getUrl()` always being a function?